esp/esp/program/modules/forms/admincore.py

```diff
@@ -32,13 +33,21 @@ def __init__(self, program, *args, **kwargs):
     def load_data(self):
         lunch_timeslots = Event.objects.filter(meeting_times__parent_class__parent_program=self.program, meeting_times__parent_class__category__category='Lunch').distinct()
         self.initial['timeslots'] = lunch_timeslots.values_list('id', flat=True)
+        should_generate_constraints = True
+        if Tag.getTag('no_lunch_constraints'):
+            should_generate_constraints = False
+        self.initial['generate_constraints'] = should_generate_constraints
+        self.initial['autocorrect'] = should_generate_constraints
+        self.initial['include_conditions'] = should_generate_constraints
 
     def save_data(self):
         timeslots = Event.objects.filter(id__in=self.cleaned_data['timeslots']).order_by('start')
-        cg = LunchConstraintGenerator(self.program, timeslots, autocorrect=(self.cleaned_data['autocorrect'] is True), include_conditions=(self.cleaned_data['include_conditions'] is True))
+        cg = LunchConstraintGenerator(self.program, timeslots, generate_constraints=(self.cleaned_data['generate_constraints'] is True), autocorrect=(self.cleaned_data['autocorrect'] is True), include_conditions=(self.cleaned_data['include_conditions'] is True))
```

Comment:
I don't see why all of these need the `is True` given that they're `BooleanField`s which can only be `True`, `False`, or perhaps `None`.